hathor/builder/builder.py

```diff
@@ -74,7 +74,7 @@ class BuildArtifacts(NamedTuple):
 
 
 _VertexVerifiersBuilder: TypeAlias = Callable[
-    [HathorSettingsType, DifficultyAdjustmentAlgorithm, FeatureService],
```

Comment:
Maybe create a `DummyFeatureService` that will be instantiated when the feature service is disabled.

Reply:
This has been changed.

Reply:
Thanks!